qlib/workflow/expm.py

```diff
@@ -227,19 +226,8 @@ def get_exp(self, experiment_id=None, experiment_name=None):
                 if self.experiments[name].id == experiment_id:
                     return self.experiments[name]
         else:
-            print("No valid experiment is found. Please make sure the id and name are correctly given.")
+            raise Exception("No valid experiment is found. Please make sure the id and name are correctly given.")
```

Comment:
We can have a more detailed format description for  search_records. Copy moflow's docs will be OK.
```
A pandas.DataFrame of runs, where each metric, parameter, and tag
        are expanded into their own columns named metrics.*, params.*, and tags.*
        respectively. For runs that don't have a particular metric, parameter, or tag, their
        value will be (NumPy) Nan, None, or None respectively.

```
Same as Experiment.search_records